core/src/utils/helpers.ts

```diff
@@ -103,6 +103,73 @@ export const inheritAttributes = (el: HTMLElement, attributes: string[] = []) =>
   return attributeObject;
 };
 
+const ariaAttributes = [
```

Comment:
Is there a list where you got all of these from? It might be good to include the source link here if we ever need to update the list.